packages/Tabs/src/Tabs.js

```diff
@@ -29,26 +42,24 @@ const Tabs = props => {
     setActiveIndex(index);
   };
 
+  // TODO: Disabled tab items should also get focus on keyboard interaction
   // https://github.com/acl-services/paprika/issues/310
-  // Todo Disabled tab items should also get focus on keyboard interaction
   const onKeyDown = (event, currentIndex) => {
-    const tabList = React.Children.toArray(props.children)[0];
-    const enabledIndexes = tabList.props.children
-      .map((tab, index) => (tab.props.isDisabled === true ? null : index))
+    const { "Tabs.List": TabsList } = extractChildren(children, ["Tabs.List"]);
+    const tabs = TabsList.props.children.filter(child => child);
```

Comment:
is this `.filter` needed? It's not filtering on anything

Reply:
if it's to handle nulls, you can do the same thing on line 52 to keep things consistent

Reply:
Yeah, it's to filter out `nulls` and I agree `child => child != null` would be more clear 👍